oc: add page (#3439)
tldr-pages/tldr: pages/common/oc.md

# oc

> > The OpenShift Container Platform CLI exposes commands for managing your applications.

Reviewer: Perhaps this will read better?
```suggestion
> The OpenShift Container Platform CLI.
> Allows for application and container management.
```

Author: Thanks

> > More information: https://docs.openshift.com/container-platform/3.11/cli_reference/get_started_cli.html.

Reviewer 2: ```suggestion
> More information: <https://docs.openshift.com/container-platform/3.11/cli_reference/get_started_cli.html>.
```

# oc

> The OpenShift Container Platform CLI.
> Allows for application and container management.
> More information: <https://docs.openshift.com/container-platform/3.11/cli_reference/get_started_cli.html>.

- Log in to the OpenShift Container Platform server:

`oc login`

- Create a new project:

`oc new-project {{project_name}}`

- Add a new app to project:

> `oc new-app {{git_link_or_docker_hub}} --name {{app_name}}`

Reviewer 3: `git_link_or_docker_hub` looks pretty complicated. Let's choose the most common option (probably a git repo) and use {{repo_url}}
